ci/rust-build-main.bat

```diff
@@ -34,27 +34,27 @@ cargo test --target %TARGET% || exit /B
 @echo Test (release)
 @echo --------------
 cargo test --target %TARGET% --release || exit /B
+@echo
+@echo Run example (release)
+@echo ---------------------
+cargo run --example builders --target %TARGET% --release
+cargo run --example dynamic_types --target %TARGET% --release
+cargo run --example read_csv --target %TARGET% --release
 
 @echo ===================================
-@echo Build with nightly toolchain
+@echo Build with stable toolchain
 @echo ===================================
 
-rustup default nightly
+rustup default stable
```

Comment:
Curious: will the build fail if it fails to build in stable? how is this done?

Reply:
I think it will right now (failure in stable will fail the build).  @andygrove I don't think you needed to add the `|| exit /B` below.  This is the mechanism that fails the build AFAIK.  Actually, I think we need to add the `|| exit /B` to each of the examples above so that the build fails if they fail to run. 

@kszucs could you comment on this?

Reply:
I have addressed this now. I had originally swapped the order of stable/nightly and that made the PR confusing to review. I added the `|| exit /B` to the examples so it is consistent.